lib/cinema/repositories/cinema_topic_repository.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:myecl/tools/repository/repository.dart';
+
+class CinemaTopicRepository extends Repository {
+  @override
+  // ignore: overridden_fields
+  final ext = 'notification/';
+  final prefix = "cinema_";
+
+  Future<bool> subscribeSession(String topic) async {
+    // return await create({}, suffix: "topics/$prefix$topic/subscribe");
```

Comment:
Why is this line commented ?

Reply:
It was to mock an API response. This should not be commented anymore